.github/workflows/ci.yml

```diff
@@ -1,33 +1,54 @@
-name: Ubuntu CI
+name: ROS 2 CI
 
-on: [pull_request]
+on:
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 0 * * *'  # every day to test Jazzy packages updates
 
 jobs:
-  test-humble-garden:
+  build-and-test:
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distribution: [jazzy]
+        include:
+          - ros_distribution: jazzy
+            ubuntu_version: 22.04
```

Comment:
Should this be `24.04`. [We're saying](https://github.com/osrf/vrx/wiki/system_requirements) that this is our recommended Ubuntu version.

Reply:
Jazzy can only be installed in Ubuntu Noble and we are using the jazzy images produced by Shane, so yes, we were using Noble. The 22.04 refers to the runner-system, I removed it to make clearer in [f77d96b](https://github.com/osrf/vrx/pull/810/commits/f77d96b8cf22ac68aa0410c53ee806ab1443a499)